### PR TITLE
Try to fix CI - Set polyfill to true in .babelrc

### PR DESCRIPTION
**Pull request by igor-dv**
Issue: #82

### CODE REVIEW

**Comment by igor-dv** [comment]
I've also reduced the version of istanbul-instrumenter-loader to 0.2.0

because of this:

image

https://travis-ci.org/alexcurtis/react-treebeard/builds/254942992

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-treebeard",
-  "version": "2.0.2",
+  "version": "2.0.3",
```
Comment (MichaelDeBoey) [inline package.json]: Please don't increment the version number on every PR.
I'd like to do it in it's own commit.

That way we can squash multiple PR's/commits into the same release

Comment (MichaelDeBoey) [inline package.json]: Please don't increment the version number on every PR.
I'd like to do it in it's own commit.

That way we can squash multiple PR's/commits into the same release
